src/style.css

```diff
@@ -2,0 +1,115 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc) format('truetype');
+}
+
+body {
+  margin: 0;
+}
+
+.search__form {
+  display: block;
+  font-family: Avenir, serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search__big {
+  margin: 20px 8px;
+  height: 70px;
+}
+
+.search__big :first-child {
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  background-image: url(./images/Search.svg);
+  background-size:
+    19px 19px,
+    cover;
+  background-repeat: no-repeat;
+  background-position:
+    25px center,
+    top;
+  border: 1px solid #e1e7ed;
+  border-radius: 5px;
+  font-family: inherit;
+  font-size: 16px;
+  font-weight: 300;
+  color: #3d4e61;
+  padding-left: 62px;
+  outline: none;
+}
+
+.search__big :first-child::placeholder {
+  font-family: Avenir, serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search__big :first-child:hover {
+  box-shadow: 0 3px 8px 0 #3d4e611a;
+}
+
+.search__big :first-child:focus {
+  box-shadow: 0 3px 8px 0 #3d4e611a;
+  border-radius: 4px;
+  background-image: url(./images/Search.svg),
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  text-shadow: 0 4px 4px #00000040;
+  border: 1px solid #e1e7ed;
+  font-family: inherit;
+  font-size: 14px;
+  font-weight: 900;
+  color: #3d4e61;
+}
+
+.search__small {
+  height: 42px;
+  margin: 0 8px;
+}
+
+.search__small :first-child {
```

Comment:
Similar to the previous comment, the ':first-child' selector may not work as intended if the input is not the first child of its parent. Consider using a more specific class or ID selector to ensure the correct element is styled.